spring-integration-cassandra/src/main/resources/org/springframework/integration/config/xml/spring-integration-cassandra-1.0.xsd

```diff
@@ -17,5 +17,187 @@
+					<xsd:choice minOccurs="0" maxOccurs="2">
+					<xsd:element name="parameter-expression" type="queryParameterType" minOccurs="0" maxOccurs="unbounded">
+					<xsd:annotation>
+						<xsd:documentation>
+							Specify an expression for parameter variable placeholder in cql statement.
+						</xsd:documentation>
+					</xsd:annotation>
+				    </xsd:element>
+					<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
+					</xsd:choice>
+					<xsd:attribute name="reply-channel" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								Message Channel to which replies should be sent after being received from Cassandra cluster.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type
+										type="org.springframework.messaging.MessageChannel"/>
+								</tool:annotation>
+							</xsd:appinfo>
+						</xsd:annotation>
+					</xsd:attribute>
+					<xsd:attribute name="input-channel" type="xsd:string">
+						<xsd:annotation>
+							<xsd:documentation>
+								Message Channel to which Messages should be sent to Cassandra.
+							</xsd:documentation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type
+										type="org.springframework.messaging.MessageChannel"/>
+								</tool:annotation>
+							</xsd:appinfo>
+						</xsd:annotation>
+					</xsd:attribute>
+				</xsd:extension>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	
+	<xsd:complexType name="outboundType">
+		<xsd:annotation>
+			<xsd:documentation>
+				Common configuration for cassandra adapters.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:attributeGroup ref="integration:channelAdapterAttributes" />
+		<xsd:attribute name="cassandra-template" type="xsd:string">
+			<xsd:annotation>
+			<xsd:documentation>
+						<![CDATA[
+							Reference to an instance of
+							org.springframework.data.cassandra.core.CassandraOperations]]>
+						</xsd:documentation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<tool:expected-type
+							type="org.springframework.data.cassandra.core.CassandraOperations" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="write-options" type="xsd:string">
+			<xsd:annotation>
+			<xsd:documentation>
+						<![CDATA[
+							Reference to an instance of
+							org.springframework.cassandra.core.WriteOptions]]>
+						</xsd:documentation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<tool:expected-type
+							type="org.springframework.cassandra.core.WriteOptions" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="type" default="INSERT" use="optional">
+			<xsd:annotation>
+				<xsd:documentation>
+				<![CDATA[
+					Indicates the operation to do on cassandra db]]>
```

Comment:
Well, actually it isn't operation on the Cassandra. It's really the style of `CassandraMessageHandler` behavior.
So, I'd really rename it to the `mode` and specify description like:
```
Indicates the `CassandraMessageHandler` behavior. Ignored in case of explicit 'query', 'ingest-query' or `statement-expression`.
```
And yes, let's consider to rename the `queryType` property to the `mode` as well!